cmd/skupper/skupper_link.go

```diff
@@ -34,6 +34,7 @@ func NewCmdLinkCreate(skupperClient SkupperLinkClient, flag string) *cobra.Comma
 	}
 	cmd.Flags().StringVarP(&connectorCreateOpts.Name, flag, "", "", "Provide a specific name for the link (used when deleting it)")
 	cmd.Flags().Int32VarP(&connectorCreateOpts.Cost, "cost", "", 1, "Specify a cost for this link.")
+	cmd.Flags().IntVarP(&connectorCreateOpts.Dcc, "dataConnectionCount", "", 1, "How many inter-router data connections")
```

Comment:
Other multi word flags use hyphens rather than camel case, so this so follow that pattern for consistency. I think the default should be 0, and that should be treated as unset, in which case a default would be applied by some other part of the code.